src/tce-core/PreviewOverlay.vue

```diff
@@ -0,0 +1,26 @@
+<template functional>
```

Comment:
`functional` attribute is removed in Vue 3, so we might as well remove it here to reduce necessary number of changes for migration.